actioncable/lib/action_cable/engine.rb

```diff
@@ -31,6 +32,10 @@ class Railtie < Rails::Engine # :nodoc:
           self.cable = Rails.application.config_for(config_path).with_indifferent_access
         end
 
+        self.connection_class = ApplicationCable::Connection
```

Comment:
Killed the error message since from the point of the final release the file with this constant has been generated.